scripts/deploy_and_verify_raft.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# $1: the name of the .env file to use
+source $1
+forge create src/Raft.sol:Raft --constructor-args $OWNER_ADDRESS $NAME $SYMBOL --private-key $PRIVATE_KEY --verify --rpc-url $ETH_RPC_URL --etherscan-api-key $ETHERSCAN_API_KEY
```

Comment:
Similar to convention followed in the other script, I'd recommend using $RAFT_NAME $RAFT_SYMBOL $RAFT_OWNER_ADDRESS

Also maybe it's more prescriptive to say $DEPLOYER_PRIVATE_KEY